packages/components/select-v2/src/useSelect.ts

```diff
@@ -468,7 +468,7 @@ const useSelect = (props: ExtractPropTypes<typeof SelectProps>, emit) => {
     }
   }
 
-  const handleBlur = () => {
+  const handleBlur = (event: Event) => {
```

Comment:
Maybe this should be `Focusevent`.

Reply:
Yes, I have fixed it. Please review again.